test/test_using_plams/test_package_wrapper.py

```diff
@@ -0,0 +1,90 @@
+import os
+import shutil
+import warnings
+from typing import Type
+from os.path import isdir, isfile
+from pathlib import Path
+
+import pytest
+from noodles.interface import PromisedObject
+from scm.plams import ADFJob, AMSJob, from_smiles
+from scm.plams.core.basejob import Job
+
+from qmflows import PackageWrapper, run, Settings
+from qmflows.utils import InitRestart
+from qmflows.packages.packages import Result
+from qmflows.packages.SCM import ADF_Result
+from qmflows.packages.package_wrapper import ResultWrapper
+
+PATH = Path('test') / 'test_files'
+ADF_ENVIRON = frozenset({'ADFBIN', 'ADFHOME', 'ADFRESOURCES', 'SCMLICENSE'})
+HAS_ADF: bool = ADF_ENVIRON.issubset(os.environ.keys())
+
+
+def _get_result(promised_object: PromisedObject, job: Type[Job]) -> Result:
```

Comment:
Continuation of https://github.com/SCM-NV/qmflows/pull/149#discussion_r387583432:
> Thanks for the info, I'll get back to you a bit later.

Allright, I've created a way to (sort of) mock the ADF results.
More specifically, it creates a normal-ish ``qmflows.Result`` instance and then attaches a pickled .dill file from a previous calculation.